packages/http-client-java/generator/pom.xml

```diff
@@ -16,22 +16,34 @@
     </properties>
 
     <modules>
-      <module>http-client-generator</module>
-      <module>http-client-generator-mgmt</module>
-      <module>http-client-generator-core</module>
+        <module>http-client-generator</module>
+        <module>http-client-generator-mgmt</module>
+        <module>http-client-generator-core</module>
     </modules>
 
     <profiles>
-      <profile>
-        <id>test</id>
-        <activation>
-          <property>
-            <name>test</name>
-          </property>
-        </activation>
-        <modules>
-          <module>http-client-generator-test</module>
-        </modules>
-      </profile>
+        <profile>
+            <id>test</id>
+            <activation>
+                <property>
+                    <name>test</name>
+                </property>
+            </activation>
+            <modules>
+                <module>http-client-generator-test</module>
+            </modules>
+        </profile>
     </profiles>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>3.2.5</version>
```

Comment:
Use surefire 3.2.5.

Otherwise the plugin used in CI is 2.12.4 (probably due to old version of maven?).